Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -171,6 +171,10 @@ def bump_formula_pr
     new_mirror ||= args.mirror || case new_url
     when requested_spec != :devel && %r{.*ftp.gnu.org/gnu.*}
       new_url.sub "ftp.gnu.org/gnu", "ftpmirror.gnu.org"
+    when %r{.*download.savannah.gnu.org/*}
+      new_url.sub "download.savannah.gnu.org", "download-mirror.savannah.gnu.org"
+    when %r{.*www.apache.org/dyn/closer.cgi*}
+      new_url.sub "www.apache.org/dyn/closer.cgi?path=", "archive.apache.org/dist/"
```

Comment:
Sometimes we have `www.apache.org/dyn/closer.lua?path="` ( `closer.lua` instead of `closer.cgi`), don't know the difference, but probably we could handle both of them

Reply:
Or if not: a `brew audit` to be consistent would be nice.

Reply:
Added an audit to prefer the Lua version since it's [mentioned here](http://www.apache.org/dev/release-download-pages.html) and shows as being [recently modified](https://www.apache.org/dyn/).